doc/api/child_process.markdown

```diff
@@ -622,13 +411,15 @@ index corresponds to a fd in the child.  The value is one of the following:
 2. `'ipc'` - Create an IPC channel for passing messages/file descriptors
    between parent and child. A ChildProcess may have at most *one* IPC stdio
    file descriptor. Setting this option enables the ChildProcess.send() method.
-   If the child writes JSON messages to this file descriptor, then this will
-   trigger ChildProcess.on('message').  If the child is an Node.js program, then
-   the presence of an IPC channel will enable process.send() and
-   process.on('message').
-3. `'ignore'` - Do not set this file descriptor in the child. Note that Node.js
-   will always open fd 0 - 2 for the processes it spawns. When any of these is
-   ignored Node.js will open `/dev/null` and attach it to the child's fd.
+   If the child writes JSON messages to this file descriptor, the
+   `ChildProcess.on('message')` event handler will be triggered in the parent.  
+   If the child is a Node.js process, the presence of an IPC channel will enable
+   `process.send()`, `process.disconnect()`, and `process.on('message')` within
```

Comment:
also, re: unref(), children start off with the control channel unrefed until process.on('disconnect') is called, then it gets refed. Also, I think when process.send() is called. Sorry, I'm going from memory here. Not sure where that info belongs, maybe in the docs for process.on('disconnect')?